sound/soc/bcm/bcm2835-i2s.c

```diff
@@ -730,7 +730,9 @@ static struct snd_soc_dai_driver bcm2835_i2s_dai = {
 	.playback = {
 		.channels_min = 2,
```

Comment:
You should apply the same changes to .capture. bcm2835-i2s rates are symmetric so the limits/constraints should be as well.